stdlib/universe/fill.gen.go.tmpl

```diff
@@ -49,3 +50,45 @@ func (t *fillTransformation) fill{{.Name}}Column(arr *{{.ArrowType}}, fillValue
 	return b.NewArray()
 }
 {{end}}
+
+func (t *fillTransformation) fillColumnNT(typ flux.ColType, arr array.Interface, fillValue *interface{}, mem arrowmem.Allocator) array.Interface {
+    switch typ {
+    {{range .}}case {{.ColumnType}}:
+        return t.fill{{.Name}}ColumnNT(arr.(*{{.ArrowType}}), fillValue, mem)
+    {{end}}
+    default:
+        panic(fmt.Errorf("unsupported array data type: %s", arr.DataType()))
+    }
+}
+
+{{range .}}
+func (t *fillTransformation) fill{{.Name}}ColumnNT(arr *{{.ArrowType}}, fillValue *interface{}, mem arrowmem.Allocator) array.Interface {
```

Comment:
I can see that the only difference between these methods and the original ones is that there is an allocator passed in here. Instead of duplicating this, can you add an allocator parameter to the old methods?

Reply:
Same as above, this is a temporary change and the original will be completely deleted.
I do not have any preference here, I can change it as suggested. Kindly let me know.